ChildProject/converters.py

```diff
@@ -472,6 +472,7 @@ def convert(filename: str, filter=None) -> pd.DataFrame:
                     "vcm_type": "NA",
                     "lex_type": "NA",
                     "mwu_type": "NA",
+                    "msc_type": "NA",
```

Comment:
open question: should it be NA by default (in which case the tests need some adaptation), or simply undefined?

Reply:
I'm not sure to understand the difference between the two.
Undefined means that the column will not appear if the "msc" tier isn't in the eaf file ?

Reply:
Yes, exactly!

Reply:
In that case I feel that the undefined way is better - the "msc_type" column doesn't exist in the already imported annotations